File: master_python_fullstack/07-ejercicios/ejercicio3.py
"""
Ejercicio 3. Escribir un programa que muestre los cuadrados 
(un número multiplicado por si mismo) de los 60 primeros numeros
naturales. Resolverlo con for y while.
"""

# WHILE
print("### Bucle WHILE ###")
contador = 0

while contador <= 60:
    
    cuadrado = contador * contador
    print(f"El cuadrado de {contador} es {cuadrado}")
    contador += 1

# FOR
print("\n### Bucle FOR ###")
for numero in range(61):
    cuadrado = numero * numero
    print(f"El cuadrado de {numero} es { cuadrado}")

File: ProgramacionBasicaPHP/ejercicio3.php
<h2>Ejercicio 3</h2>
<?php
/*Escribe un programa que imprima por pantalla 
los cradrados(el número multiplicado por sí mismo)
de los 30 primeros números naturales.

Modifica el ejercicio anterior para que muestre al
lado de cada cuadrado si es un número para o impar.
*/

for($i = 1; $i <= 30; $i++) {

    $cuadrado = $i * $i;

    echo "El cuadrado de ".$i.' es '.$cuadrado;

    if($cuadrado % 2 == 0){
        echo " es par";
    }else{
        echo " es impar";
    }

    echo "<br/>";
}
?>